spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe PushmiPullyu::SwiftDepositer do
+  it 'not valid with nil arguments' do
+    expect { described_class.new(nil, nil) }.to raise_error(RuntimeError)
+  end
+
+  it 'valid connection established' do
```

Comment:
You could pretty much combine this spec with the one below. As they are very similar. Both are creating an instance of PushmiPullyu::SwiftDepositer. Can just move the `expect()` from this one down into the other spec. Then everything you wanted to test is still done, without all this extra code (and another VCR tape).